.devcontainer/installMoreTools.sh

```diff
@@ -6,3 +6,5 @@ sudo apt-get update && sudo apt-get install -y lsb-release wget software-propert
 # Install LLVM 14
 export LLVM_VERSION=14
 curl -sL https://apt.llvm.org/llvm.sh | sudo bash -s "$LLVM_VERSION"
+echo 'export PATH=$PATH:/usr/lib/llvm-14/bin' >> ~/.profile
```

Comment:
this differs from how we install llvm in the bulid containers, should we make them consistent?

Reply:
should the devcontainer be the same base mariner image as the build container?

Reply:
hm im actually not too familiar with the pros and cons, i was thinking jammy would be more known and easy to develop on, but i dont have much experience with mariner. I can update this if you think it is better, then we can be more consistent

Reply:
I think that the closer the dev / prod impedence match is, the more likely you are to catch compatibility issues. I'd say go with mariner if it's usable in dev.

Reply:
I did mess around for a little bit with this image, seems that some devcontainer features (docker-in-docker, kind, kubernetes-cli, and github-cli) don't work by default so we would have to set that up 
Im not sure if it's worth it to figure out the configuration for the above missing features 
If you guys think it's worth it then i can play around with it a bit more and figure something out

In this current setup, there is docker-in-docker and kind which works -> if we were to use mariner im not sure if this would work - im thinking it would need to mount to a socket?

I was just envisioning it as installing dependencies, running out make commands that generate retina images, and deploying to kind cluster, so i don't see as much importance in matching this container image with the retina container image